### PR TITLE
Move tests to GithubActions

### PR DESCRIPTION
**Pull request by schneems**
All tests ported except for YJIT

```
      - run: echo "export RUBY_YJIT_ENABLE=1" >> $BASH_ENV
```

### CODE REVIEW

**Comment by schneems** [comment]
Getting a failure https://github.com/zombocom/syntax_suggest/actions/runs/2770136464:

```
  bundle exec standarb
  shell: /usr/bin/bash -e {0}
##[debug]/usr/bin/bash -e /home/runner/work/_temp/1df3154b-3045-4a10-b1c9-3c48c5bc1cc6.sh
bundler: command not found: standarb
Install missing gem executables with `bundle install`
Error: Process completed with exit code 127.
```

However in `bundle install` it shows as installed:

```
  ##[debug]Downloading segment at offset 0 with length 5619027...
  Received 5619027 of 5619027 (100.0%), 28.7 MBs/sec
  /usr/bin/tar --use-compress-program zstd -d -tf /home/runner/work/_temp/45297a7a-04e7-48c0-9eef-cbbedf748685/cache.tzst -P
  vendor/bundle/
  vendor/bundle/ruby/
  vendor/bundle/ruby/3.1.0/
  vendor/bundle/ruby/3.1.0/bin/
  vendor/bundle/ruby/3.1.0/bin/ruby-rewrite
  vendor/bundle/ruby/3.1.0/bin/syntax_suggest
  vendor/bundle/ruby/3.1.0/bin/rake
  vendor/bundle/ruby/3.1.0/bin/ldiff
  vendor/bundle/ruby/3.1.0/bin/rspec
  vendor/bundle/ruby/3.1.0/bin/standardrb
```

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - name: Linting
+        run: bundle exec standarb
```
Comment (tachyons) [inline .github/workflows/ci.yml:30]:
```suggestion
        run: bundle exec standardrb
```

**Review by tachyons — CHANGES_REQUESTED** [review]
You made a typo here

File: .github/workflows/ci.yml (hunk 1 above)
Comment (schneems) [reply .github/workflows/ci.yml]: ❤️

**Pull request merged**